src/components/Header.tsx

```diff
@@ -39,25 +53,48 @@ const Header = (props: BoxProps) => {
                 textDecorationColor: 'primary.900',
               }}
             >
-              <HStack spacing={2}>
+              <HStack spacing={{ base: 1, lg: 2 }}>
                 <Box h={LOGO_SIZES} w={LOGO_SIZES}>
-                  <Image src={mascotIcon} alt="" height={150} width={150} />
+                  <Image
+                    src={mascotIcon}
+                    alt=""
+                    height={70}
+                    width={70}
+                    aria-hidden
+                  />
                 </Box>
                 <Heading
                   as="h1"
                   color="primaryText"
                   letterSpacing="tight"
                   fontWeight="black"
-                  fontSize={{ base: '16px', sm: '36px' }}
+                  fontSize={{ base: '16px', md: '36px' }}
                 >
                   Octoclairvoyant
                 </Heading>
               </HStack>
             </RouteLink>
           </Flex>
-          {!!process.env.NEXT_PUBLIC_FEATURE_FLAG_COLOR_MODE && (
-            <ToggleColorModeButton />
-          )}
+          <HStack spacing={{ base: 4, md: 10 }}>
+            {!!process.env.NEXT_PUBLIC_FEATURE_FLAG_COLOR_MODE && (
+              <ToggleColorModeButton
+                boxSize={LOGO_SIZES}
+                minWidth={LOGO_SIZES}
+              />
+            )}
+            <Link
+              href={REPO_URL}
+              aria-label="Octoclairvoyant repository on GitHub"
+              isExternal
+            >
+              <Icon
+                as={DiGithubBadge}
+                boxSize={LOGO_SIZES}
+                color="monochrome1"
+                _hover={{ rounded: 'full', bgColor: 'monochromeInverted4' }}
```

Comment:
Not really sure what to do with the GitHub icon on hover.

Reply:
Hmm, I'll go think about that a bit. 😉